x/sequencer/keeper/grpc_query_sequencers_by_rollapp.go

```diff
@@ -60,11 +65,16 @@ func (k Keeper) GetNextProposerByRollapp(c context.Context, req *types.QueryGetN
 	}
 	ctx := sdk.UnwrapSDKContext(c)
 
-	successor := k.GetSuccessor(ctx, req.RollappId)
 	inProgress := k.AwaitingLastProposerBlock(ctx, req.RollappId)
 
+	addr := ""
+	successor := k.GetSuccessor(ctx, req.RollappId)
+	if !successor.Sentinel() {
+		addr = successor.Address
+	}
+
 	return &types.QueryGetNextProposerByRollappResponse{
-		NextProposerAddr:   successor.Address,
+		NextProposerAddr:   addr,
```

Comment:
afaiu this only changes from returning `sentinel` to return `` - which is still invalid address. whats the purpose of this?